api/swagger.yaml

```diff
@@ -7202,6 +7202,11 @@ paths:
             - `reference`=(`<image-name>[:<tag>]`)
             - `since`=(`<image-name>[:<tag>]`,  `<image id>` or `<image@digest>`)
           type: "string"
+        - name: "shared-size"
```

Comment:
The API and docs changes should go into the same commit as the code changes (as one shouldn't get in without the other).